src/igvfd/tests/test_schema_disease_ontology_term.py

```diff
@@ -0,0 +1,9 @@
+import pytest
+
+
+def test_disease_ontology_term_term_id_regex(disease_ontology_term_alzheimers, testapp):
```

Comment:
I think you can also add a test for required properties
I also think the inverse of regex failure could be executed (even though inserts cover that)